assets/components/directDebit/directDebitForm/directDebitForm.jsx

```diff
@@ -156,63 +190,124 @@ function AccountNumberInput(props: {onChange: Function, value: string}) {
-      <input
-        id="account-holder-name-input"
-        value={props.value}
-        onChange={props.onChange}
-        maxLength="18"
-        className="component-direct-debit-form__text-field focus-target"
-      />
+      {props.phase === 'entry' ? editable : locked}
     </div>
   );
 }
 
-function ConfirmationInput(props: { checked: boolean, onChange: Function }) {
+function ConfirmationInput(props: {phase: string, checked: boolean, onChange: Function }) {
+  const editable = (
+    <span>
+      <div className="component-direct-debit-form__confirmation-css-checkbox">
+        <input
+          className="component-direct-debit-form__confirmation-input"
+          id="confirmation-input"
+          type="checkbox"
+          onChange={props.onChange}
+          checked={props.checked}
+        />
+        <label
+          className="component-direct-debit-form__confirmation-label"
+          htmlFor="confirmation-input"
+        />
+      </div>
+      <span className="component-direct-debit-form__confirmation-text">
+        I confirm that I am the account holder and I am solely able to authorise debit from
+        the account
+      </span>
+    </span>
+  );
+
+  const locked = (
+    <span>
+      <label htmlFor="confirmation-text__locked" className="component-direct-debit-form__field-label">
+        Declaration
+      </label>
+      <div id="confirmation-text__locked" className="component-direct-debit-form__confirmation-text__locked">
+        I have confirmed that I am the account holder and that I am solely able to authorise debit
+        from the account
+      </div>
+      <div className="component-direct-debit-form__confirmation-guidance">
+        If the details above are correct press confirm to set up your direct debit, otherwise press
+        edit to go back and make changes
+      </div>
+    </span>
+  );
+
   return (
     <div className="component-direct-debit-form__account-holder-confirmation">
       <div>
         <label htmlFor="confirmation-input">
-          <div className="component-direct-debit-form__confirmation-css-checkbox">
-            <input
-              className="component-direct-debit-form__confirmation-input"
-              id="confirmation-input"
-              type="checkbox"
-              onChange={props.onChange}
-              checked={props.checked}
-            />
-            <label
-              className="component-direct-debit-form__confirmation-label"
-              htmlFor="confirmation-input"
-            />
-          </div>
-          <span className="component-direct-debit-form__confirmation-text">
-            I confirm that I am the account holder and I am solely able to authorise debit from
-            the account
-          </span>
+          {props.phase === 'entry' ? editable : locked}
         </label>
       </div>
     </div>
   );
 }
 
-function PaymentButton(props: {onClick: Function}) {
-  return (
-    <button
-      id="qa-pay-with-direct-debit-pay"
```

Comment:
This appears to be the first element in this component to be given a QA id. Are there any post-deploy tests that exist for direct debit? If not, could we drop the ids for now, and then add them in future if they're needed @svillafe?

Reply:
I can confirm there isn't any post-deploy test for direct debit. We should add them.